inbm/dispatcher-agent/dispatcher/sota/sota.py

```diff
@@ -125,7 +125,7 @@ def __init__(self,
             # If an exception occurs during string conversion, raise that exception
             raise SotaError('package_list is not a string in manifest') from e
 
-        self._reboot_device = parsed_manifest['deviceReboot']        
+        self._reboot_device = parsed_manifest['deviceReboot']
```

Comment:
**<h2>$\color{red}Defects: ErrorHandling$</h2>**
**Suggestion:** Add a check to ensure 'deviceReboot' key exists in parsed_manifest to avoid KeyError.
**NOTE:** Ensure that the suggested code is comprehensive. This serves as an example of the code you could use. The user should verify it after accepting the suggestion.
```suggestion
        self._reboot_device = parsed_manifest.get('deviceReboot', 'Yes')
```

Reply:
good suggestion, but out of scope for code formatting change